src/routes/deckShare/[sharedKey]/+page.svelte

```diff
@@ -25,4 +25,20 @@
 
 <svelte:head>
   <title>Deck Share</title>
-</svelte:head>
\ No newline at end of file
+</svelte:head>
+
+<style>
+  button {
+    font-size: large;
+    font-weight: bold;
+    margin-top: 0;
+  }
```

Comment:
This should fix the button text thing

Reply:
yup just checked on safari and it looks good!